packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/Form/index.js

```diff
@@ -55,13 +69,47 @@ const Form = forwardRef(({ projectSettingsStored }, ref) => {
           defaultMessage: 'Customization',
         })}
       </Typography>
-      <Grid paddingTop={4}>
+      <Grid paddingTop={4} gap={4}>
         <GridItem col={6} s={12}>
           <LogoInput
-            onChangeLogo={handleChangeMenuLogo}
+            canUpdate={canUpdate}
             customLogo={menuLogo.display}
             defaultLogo={menu.default}
-            onResetMenuLogo={handleResetMenuLogo}
+            hint={formatMessage(
+              {
+                id: 'Settings.application.customization.menu-logo.carousel-hint',
+                defaultMessage:
+                  'Change the admin panel logo (Max dimension: {dimension}x{dimension}, Max file size: {size}KB)',
```

Comment:
As we now have potentially two different logos, I'd rework a little bit the content. For this one, I'd say "Replace the logo in the main navigation"

And for the other one: "Change the logo on the authentication page"

Of course, the part between ( ) must remain. What do you think?

Reply:
sounds really good, I would just make page plural "_Change the logo on the authentication pages_" since there's the log in, reset password, lost password pages - wdyt?

Reply:
the description/hint and label placements is handled by the DS as all other inputs 👀 @maevalienard

Reply:
wait it just clicked sorry, below the Customization title? I can surely do that!